README.md

```diff
@@ -4,30 +4,312 @@
+
+# Testing mode (dry run)
+# Set testing: True in config file
+sync2jira
+```
+
+### Manual Sync Interface
+
+Access the sync page at `http://your-sync-page-url` to manually trigger synchronization for specific repositories.
+
+### Environment Variables
+
+- `INITIALIZE`: Set to `1` to perform initial sync of all issues
+- `FEDORA_MESSAGING_QUEUE`: Custom message queue name
+- `GITHUB_TOKEN`: GitHub API token (can override config)
+- `JIRA_TOKEN`: JIRA API token (can override config)
+
+## Development
+
+### Setting up Development Environment
+
+```bash
+# Clone and install in development mode
+git clone https://github.com/release-engineering/Sync2Jira.git
+cd Sync2Jira
+pip install -e .
+
+# Install development dependencies
+pip install -r test-requirements.txt
+```
+
+### Running Tests
+
+```bash
+# Run all tests
+tox
+
+# Run specific test environments
+tox -e py39          # Python 3.9 tests
+tox -e lint          # Linting
+tox -e black         # Code formatting check
+tox -e isort         # Import sorting check
+
+# Run tests with coverage
+coverage run -m pytest
+coverage report
+```
+
+### Code Quality
+
+This project uses several tools to maintain code quality:
+
+- **Flake8**: Linting and style checking
+- **Black**: Code formatting
+- **isort**: Import sorting
+- **pytest**: Testing framework
+- **Coverage**: Test coverage reporting
+
+## Documentation
+
+Comprehensive documentation is available at [sync2jira.readthedocs.io](https://sync2jira.readthedocs.io/en/latest/):
+
+- [Quick Start Guide](https://sync2jira.readthedocs.io/en/master/quickstart.html)
+- [Configuration Reference](https://sync2jira.readthedocs.io/en/master/config-file.html)
+- [Adding New Repositories](https://sync2jira.readthedocs.io/en/master/adding-new-repo-guide.html)
+- [Sync Page Usage](https://sync2jira.readthedocs.io/en/master/sync_page.html)
+
+## Contributing
+
+We welcome contributions! Please see our [CONTRIBUTING.md](CONTRIBUTING.md) for details on:
+
+- Setting up your development environment
+- Running tests
+- Submitting pull requests
+- Coding standards
+
+## Common Use Cases
+
+### Syncing Issues Only
+
+```python
+'map': {
+    'github': {
+        'org/repo': {
+            'project': 'PROJ',
+            'sync': ['issue'],  # Only sync issues
+            'issue_updates': ['title', 'description', 'comments']
+        }
+    }
+}
+```
+
+### Custom Field Mapping
+
+```python
+'map': {
+    'github': {
+        'org/repo': {
+            'project': 'PROJ',
+            'custom_fields': {
+                'customfield_10001': 'Static Value',
+                'customfield_10002': '[remote-link]'  # Will be replaced with GitHub URL
+            }
+        }
+    }
+}
+```
+
+### Filtering by Labels
+
+```python
+'filters': {
+    'github': {
+        'org/repo': {
+            'labels': ['bug', 'enhancement'],  # Only sync issues with these labels
+            'state': 'open'  # Only sync open issues
+        }
+    }
+}
+```
+
+## Troubleshooting
+
```

Comment:
Ditto previous comment:  is this covered elsewhere?...if so, we don't want it here.

Reply:
I still recommend removing this section.